src/ga4gh/vrs/extras/vcf_annotation.py

```diff
@@ -102,10 +102,17 @@ class SeqRepoProxyType(str, Enum):
     show_default=True,
     help="Skip VRS computation for REF alleles."
 )
+@click.option(
+    "--require_validation",
+    is_flag=True,
+    default=True,
+    show_default=True,
+    help="Require validation checks to pass in order to return a VRS object"
+)
 def annotate_click(  # pylint: disable=too-many-arguments
     vcf_in: str, vcf_out: Optional[str], vrs_pickle_out: Optional[str],
     vrs_attributes: bool, seqrepo_dp_type: SeqRepoProxyType, seqrepo_root_dir: str,
-    seqrepo_base_url: str, assembly: str, skip_ref: bool
+    seqrepo_base_url: str, assembly: str, skip_ref: bool, require_validation: bool
```

Comment:
do we need to pass require_validation through to annotator.annotate below as well?